app/controllers/conference_controller.rb

```diff
@@ -16,6 +16,9 @@ def schedule
     @events = @conference.program.events
     @dates = @conference.start_date..@conference.end_date
 
+    @scheduled_events = @conference.program.events.scheduled.order(:start_time)
+    @unscheduled_events = @conference.program.events.unscheduled
```

Comment:
@hennevogel wrong! That was the point why we implemented the "All Events" tab in the first place. Because we wanted to show the accepted and confirmed talks even if they aren't scheduled yet!